### PR TITLE
Better Lane Arrows Buttons

### PR DESCRIPTION
**Pull request by kvakvs**
Modifies Lane Arrow GUI to contain larger buttons with graphics for ON and OFF for each direction.
P.S. Blue text was temporary it is white again.

image

### CODE REVIEW

**Comment by FireController1847** [comment]
Is it just me or do these buttons look really tacky?

**Comment by kvakvs** [comment]
@FireController1847 you're welcome to suggest better graphics. I am not 100% happy with the disabled lane arrows.

**Comment by kvakvs** [comment]
image

**Comment by aubergine10** [comment]
The second image (comment above) looks vastly improved, and hugely better than current TMPE UI for that feature. Do we need the lane numbers? It's pretty obvious that there is a set of arrows per lane.

File: TLM/TLM/UI/TextureResources.cs [hunk 1]
```diff
@@ -42,6 +43,14 @@ public class TextureResources
         public static readonly Texture2D PedestrianModeAutomaticTexture2D;
         public static readonly Texture2D PedestrianModeManualTexture2D;
 		public static readonly IDictionary<PriorityType, Texture2D> PrioritySignTextures;
```
Comment (aubergine10) [inline TLM/TLM/UI/TextureResources.cs]: Minor: Weird indenting in this file.

Comment (kvakvs) [reply TLM/TLM/UI/TextureResources.cs]: In Speed Limits PR #384 this file is formatted. I am trying to avoid formatting same files in multiple PR.

**Comment by kvakvs** [comment]
It turns out we absolutely don't need `"Lane" + i` titles
image

**Comment by dymanoid** [comment]
Just a thought. Can we disable/hide buttons for invalid/senseless configurations? 

Like:
lanes

If I select "middle lane forwards and right", it shouldn't be possible to select "forwards and left" for the right lane and "right" for the left lane, because intersections occur.

**Comment by kvakvs** [comment]
The buttons should not disappear. Maybe empty buttons instead?

p.s. I am working on GUI which is rendered in the ground plane, it might become a much better replacement for lane arrows GUI than this branch.

**Comment by aubergine10** [comment]
Removal of buttons looks weird. Maybe still have the arrow, greyed out, with no button background?

@kvakvs Regarding the GUI rendered on ground plane, that's certainly the direction we want to head in, but be aware there are some weird pitfalls and potential performance issues with that (see PR #282 for examples).

**Comment by kvakvs** [comment]
@aubergine10 The world space UI is rendered via Canvas, i.e. it is constructed once to be a part of the scene, not an overlay rendered every frame. Unity handles that kind of rendering better.

**Comment by aubergine10** [comment]
Is it worth closing this PR considering #391 ?

**Comment by kvakvs** [comment]
I think so, canvas are the future

**Comment by aubergine10** [comment]
Made obsolete by #391

**Pull request closed without merging**